README.md

```diff
@@ -212,37 +212,36 @@ If you maintain a community distribution and want it listed here, leave an issue
 ```console
 $ rustscan -h
 
+```
+rustscan 1.7.1
 Fast Port Scanner built in Rust. WARNING Do not use this program against sensitive infrastructure since the specified
 server may not be able to handle this many socket connections at once. - Discord https://discord.gg/GFrQsGy - GitHub
 https://github.com/RustScan/RustScan
 
 USAGE:
-    rustscan [FLAGS] [OPTIONS] <ips-or-hosts>... [-- <command>...]
+    rustscan [FLAGS] [OPTIONS] <ips>... [-- <command>...]
 
 FLAGS:
-    -a, --accessible
-    -h, --help          Prints help information
-    -q, --quiet         Quiet mode. Only output the ports. No Nmap. Useful for grep or outputting to a file
-    -V, --version       Prints version information
+    -h, --help       Prints help information
+    -q, --quiet      Quiet mode. Only output the ports. No Nmap. Useful for grep or outputting to a file
+    -V, --version    Prints version information
 
 OPTIONS:
     -b, --batch-size <batch-size>    The batch size for port scanning, it increases or slows the speed of scanning.
                                      Depends on the open file limit of your OS.  If you do 65535 it will do every port
                                      at the same time. Although, your OS may not support this [default: 4500]
-    -p, --ports <ports>...           A list of comma separed ports to be scanned. Example: 80,443,8080.
-    -r, --range <range>              A range of ports with format start-end. Example: 1-1000
         --scan-order <scan-order>    The order of scanning to be performed. The "serial" option will scan ports in
                                      ascending order while the "random" option will scan ports randomly [default:
                                      serial]  [possible values: Serial, Random]
     -t, --timeout <timeout>          The timeout in milliseconds before a port is assumed to be closed [default: 1500]
     -u, --ulimit <ulimit>            Automatically ups the ULIMIT with the value you provided
 
 ARGS:
-    <ips-or-hosts>...    A list of comma separated IP addresses or hosts to be scanned
```

Comment:
this README is out-dated, it was over-written by a merge conflict. Please update  to the latest one :)

Reply:
Done!